rules/auth0_rules/auth0_mfa_policy_enabled.yml

```diff
@@ -0,0 +1,377 @@
+AnalysisType: rule
+Description: An Auth0 User enabled MFA Policy for your organization's tenant.
+DisplayName: "Auth0 MFA Policy Enabled"
+Enabled: true
+Filename: auth0_mfa_policy_enabled.py
+Runbook: Assess if this was done by the user for a valid business reason. Be vigilant to re-enable this setting as it's in the best security interest for your organization's security posture.
+Severity: Medium
+Tests:
+    - ExpectedResult: true
```

Comment:
will you please add a test case where `ExpectedResult: false`?

Reply:
Sure!

Reply:
Done